tfhe/benches/integer/bench.rs

```diff
@@ -35,9 +35,20 @@ impl Default for ParamsAndNumBlocksIter {
             Err(_) => false,
         };
 
+        let is_fast_bench = match env::var("__TFHE_RS_FAST_BENCH") {
+            Ok(val) => val.to_lowercase() == "true",
+            Err(_) => false,
+        };
+
+        let mut bit_sizes = vec![32];
+
         if is_multi_bit {
```

Comment:
I think we can better factorize this if you have some time otherwise should be good

Reply:
What do you mean by that ?

Reply:
let bit_sizes = if is_multi_bit {
...}

to have defaults per case